### PR TITLE
changed ipython to jupyter

### PR DESCRIPTION
**Pull request by annefou**
When converting jupyter notebook to html, ipython command is used instead of jupyter
(ipython is becoming obsolete).

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks @annefou! And welcome to Galaxy!

**Comment by nsoranzo** [comment]
Thanks @annefou!
If you want to dig further, there are more occurrences of "IPython" in the files you modified and in the rest of the Galaxy codebase.
```
$ ack -i -l --ignore-dir={.venv,database,eggs,.git,doc/build} ipython .
```
gives me the following list of files:
```
config/galaxy.ini.sample
config/plugins/interactive_environments/jupyter/config/allowed_images.yml.sample
config/plugins/interactive_environments/jupyter/static/js/jupyter.js
config/plugins/interactive_environments/jupyter/templates/jupyter.mako
doc/source/admin/interactive_environments.rst
doc/source/admin/interactive_environments.svg
doc/source/dev/faq.rst
doc/source/dev/interactive_environments.rst
doc/source/releases/15.10.rst
doc/source/releases/16.04.rst
lib/galaxy/config.py
lib/galaxy/datatypes/text.py
lib/galaxy/visualization/plugins/plugin.py
lib/galaxy/visualization/plugins/registry.py
lib/galaxy/web/framework/middleware/sentry.py
lib/galaxy/web/proxy/js/Dockerfile
test/docker/base/Dockerfile
test/unit/visualizations/plugins/test_VisualizationsRegistry.py
```

But you can ignore at least `config/plugins/interactive_environments/jupyter/static/js/jupyter.js` .

If you don't have time/are not interested in fixing those too, that's perfectly fine, we can merge this pull request and open an issue to track the other files.

**Comment by annefou** [comment]
Ok, I'll look at it more carefully and see what else needs to be updated.

**Comment by jmchilton** [comment]
@galaxybot test this

File: config/galaxy.ini.sample [hunk 1]
```diff
@@ -784,8 +784,8 @@ nglims_config_file = tool-data/nglims.yaml
 # notebooks in Galaxy histories.  This process may allow users to execute
 # arbitrary code or serve arbitrary HTML.  If enabled, IPython must be
 # available and on Galaxy's PATH, to do this run
-# `pip install jinja2 pygments ipython` in Galaxy's virtualenv.
-#trust_ipython_notebook_conversion = False
+# `pip install jinja2 pygments jupyter` in Galaxy's virtualenv.
+#trust_jupyter_notebook_conversion = False
```
Comment (bgruening) [inline config/galaxy.ini.sample:788]: @jmchilton do we need to implement some fallbacks for changing config names?

**Comment by bgruening** [comment]
LGTM. One question is if we need to implement some fallback for the changed config parameter name. Hope that @jmchilton can answer this.

Thanks @annefou!

File: config/galaxy.ini.sample (hunk 1 above)
Comment (jmchilton) [reply config/galaxy.ini.sample:788]: Yup - I'll follow up with a new a PR for that.

**Comment by jmchilton** [comment]
Awesome sauce @annefou - thanks a ton for the original fix and going the extra mile and fixing it everywhere. This is really appreciated!

**Pull request merged**